test/extensions/implementation/autograd.py

```diff
@@ -57,9 +57,13 @@ def variance(self) -> List[Tensor]:  # noqa: D102
         ]
 
     def _get_diag_ggn(self, loss: Tensor, output: Tensor) -> List[Tensor]:
-        diag_ggn_flat = cat(
-            [col[[i]] for i, col in enumerate(self._ggn_columns(loss, output))]
-        )
+        try:
+            diag_ggn_flat = cat(
+                [col[[i]] for i, col in enumerate(self._ggn_columns(loss, output))]
+            )
+        # raises RuntimeError if list is empty
+        except RuntimeError:
+            diag_ggn_flat = Tensor()
```

Comment:
For which test scenarios is this expected to happen?

Reply:
Does not happen anymore. Probably some old tests of mine without any parameters. I reverted this change.

Reply:
Resolved.